features/bootstrap/FeatureContext.php

```diff
@@ -15,14 +15,16 @@
 	$project_composer = dirname( dirname( dirname( __FILE__ ) ) ) . '/composer.json';
 	$built_phar = dirname( dirname( dirname( __FILE__ ) ) ) . '/wp_launch_check.phar';
 	
+	// Load WPLC via Phar
```

Comment:
Is there a way this could be done in a Behat bootstrap or init function, rather than running this code whenever this file is included?

Reply:
I really wanted to rewrite this, but every time I attempted it, there was some weird error as to how WPCLI was being loaded and how it included Launch Check. I'll agree this is pretty brittle, but I took the approach of baby steps to just get Behat running successfully. I even made an attempt to migrate from Behat v2 -> v3, but that caused more headache, so I abandoned that in the interest of just getting Github Actions working.